? conf27324.sh
? evolution-data-server-1.2.pc.in
? addressbook/libebook/libebook-1.2.pc
? addressbook/libebook/libebook-1.2.pc.in
? addressbook/libebook/libebook.pc
? addressbook/libedata-book/libedata-book-1.2.pc
? addressbook/libedata-book/libedata-book-1.2.pc.in
? addressbook/libedata-book/libedata-book.pc
? calendar/backends/weather
? calendar/backends/http/.e-cal-backend-http.c.swp
? calendar/libecal/.e-cal.c.swp
? calendar/libecal/libecal-1.2.pc
? calendar/libecal/libecal-1.2.pc.in
? calendar/libecal/libecal.pc
? calendar/libedata-cal/libedata-cal-1.2.pc
? calendar/libedata-cal/libedata-cal-1.2.pc.in
? calendar/libedata-cal/libedata-cal.pc
? calendar/libical/src/libical/.icalenums.h.swp
? calendar/libical/src/libicalss/icalssyacc.output
? libedataserver/libedataserver-1.2.pc
? libedataserver/libedataserver-1.2.pc.in
? libedataserver/libedataserver.pc
? servers/groupwise/libegroupwise-1.2.pc.in
? servers/groupwise/libegroupwise.pc
? src/GNOME_Evolution_DataServer_1.2.server
Index: configure.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/configure.in,v
retrieving revision 1.69
diff -u -r1.69 configure.in
--- configure.in	23 Sep 2004 16:45:27 -0000	1.69
+++ configure.in	25 Sep 2004 09:12:59 -0000
@@ -312,6 +312,7 @@
 calendar/backends/groupwise/Makefile
 calendar/backends/http/Makefile
 calendar/backends/contacts/Makefile
+calendar/backends/weather/Makefile
 calendar/tests/Makefile
 calendar/tests/ecal/Makefile
 libdb/Makefile
Index: calendar/backends/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/backends/Makefile.am,v
retrieving revision 1.3
diff -u -r1.3 Makefile.am
--- calendar/backends/Makefile.am	9 Jan 2004 20:05:27 -0000	1.3
+++ calendar/backends/Makefile.am	25 Sep 2004 09:12:59 -0000
@@ -1 +1 @@
-SUBDIRS = file groupwise http contacts
+SUBDIRS = file groupwise http contacts weather
Index: calendar/libecal/e-cal-component.c
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/libecal/e-cal-component.c,v
retrieving revision 1.10
diff -u -r1.10 e-cal-component.c
--- calendar/libecal/e-cal-component.c	23 Sep 2004 21:06:46 -0000	1.10
+++ calendar/libecal/e-cal-component.c	25 Sep 2004 09:13:00 -0000
@@ -1067,7 +1067,8 @@
 	      || kind == ICAL_VTODO_COMPONENT
 	      || kind == ICAL_VJOURNAL_COMPONENT
 	      || kind == ICAL_VFREEBUSY_COMPONENT
-	      || kind == ICAL_VTIMEZONE_COMPONENT))
+	      || kind == ICAL_VTIMEZONE_COMPONENT
+	      || kind == ICAL_XWEATHER_COMPONENT))
 		return FALSE;
 
 	priv->icalcomp = icalcomp;
@@ -1170,6 +1171,9 @@
 	case ICAL_VTIMEZONE_COMPONENT:
 		return E_CAL_COMPONENT_TIMEZONE;
 
+	case ICAL_XWEATHER_COMPONENT:
+		return E_CAL_COMPONENT_XWEATHER;
+
 	default:
 		/* We should have been loaded with a supported type! */
 		g_assert_not_reached ();
Index: calendar/libecal/e-cal-component.h
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/libecal/e-cal-component.h,v
retrieving revision 1.5
diff -u -r1.5 e-cal-component.h
--- calendar/libecal/e-cal-component.h	23 Sep 2004 21:06:46 -0000	1.5
+++ calendar/libecal/e-cal-component.h	25 Sep 2004 09:13:00 -0000
@@ -48,7 +48,8 @@
 	E_CAL_COMPONENT_TODO,
 	E_CAL_COMPONENT_JOURNAL,
 	E_CAL_COMPONENT_FREEBUSY,
-	E_CAL_COMPONENT_TIMEZONE
+	E_CAL_COMPONENT_TIMEZONE,
+	E_CAL_COMPONENT_XWEATHER
 } ECalComponentVType;
 
 /* Field identifiers for a calendar component; these are used by the data model
Index: calendar/libecal/e-cal.c
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/libecal/e-cal.c,v
retrieving revision 1.75
diff -u -r1.75 e-cal.c
--- calendar/libecal/e-cal.c	27 Aug 2004 18:09:59 -0000	1.75
+++ calendar/libecal/e-cal.c	25 Sep 2004 09:13:01 -0000
@@ -214,6 +214,8 @@
 		return GNOME_Evolution_Calendar_TYPE_TODO;
 	case E_CAL_SOURCE_TYPE_JOURNAL:
 		return GNOME_Evolution_Calendar_TYPE_JOURNAL;
+	case E_CAL_SOURCE_TYPE_WEATHER:
+		return GNOME_Evolution_Calendar_TYPE_WEATHER;
 	default:
 		return GNOME_Evolution_Calendar_TYPE_ANY;
 	}
@@ -1010,19 +1012,18 @@
 	char *query;
 	int i;
 
-
 	/* Determine the protocol and query for factory supporting that */
 	uri = e_uri_new (str_uri);
 	if (!uri) {
 		g_warning (G_STRLOC ": Invalid uri string");
-		
+
 		return FALSE;
 	}
 
 	query = g_strdup_printf ("repo_ids.has ('IDL:GNOME/Evolution/DataServer/CalFactory:" BASE_VERSION "')"
 				 " AND calendar:supported_protocols.has ('%s')", uri->protocol);
 
-	
+
 	servers = bonobo_activation_query (query, NULL, NULL);
 
 	g_free (query);
@@ -1030,21 +1031,21 @@
 
 	if (!servers) {
 		g_warning (G_STRLOC ": Unable to query for calendar factories");
-		
+
 		return FALSE;
-	}	
-	
+	}
+
 	/* Try to activate the servers for the protocol */
 	for (i = 0; i < servers->_length; i++) {
 		const Bonobo_ServerInfo *info;
 
 		info = servers->_buffer + i;
 
-#if 0
+#if 1
 		g_message (G_STRLOC ": Activating calendar factory (%s)", info->iid);
 #endif
 		factory = bonobo_activation_activate_from_id (info->iid, 0, NULL, NULL);
-		
+
 		if (factory == CORBA_OBJECT_NIL)
 			g_warning (G_STRLOC ": Could not activate calendar factory (%s)", info->iid);
 		else
@@ -1359,9 +1360,9 @@
 e_cal_new (ESource *source, ECalSourceType type)
 {
 	ECal *ecal;
-	
+
 	e_cal_activate ();
-	
+
 	ecal = g_object_new (E_TYPE_CAL, NULL);
 
 	if (!fetch_corba_cal (ecal, source, type)) {
@@ -4495,6 +4496,7 @@
 {
 	switch (type) {
 	case E_CAL_SOURCE_TYPE_EVENT:
+	case E_CAL_SOURCE_TYPE_WEATHER:
 		return get_sources (sources, "/apps/evolution/calendar/sources", error);		
 		break;
 	case E_CAL_SOURCE_TYPE_TODO:
Index: calendar/libecal/e-cal.h
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/libecal/e-cal.h,v
retrieving revision 1.19
diff -u -r1.19 e-cal.h
--- calendar/libecal/e-cal.h	2 Jun 2004 16:15:00 -0000	1.19
+++ calendar/libecal/e-cal.h	25 Sep 2004 09:13:01 -0000
@@ -52,7 +52,8 @@
 	E_CAL_SOURCE_TYPE_EVENT,
 	E_CAL_SOURCE_TYPE_TODO,
 	E_CAL_SOURCE_TYPE_JOURNAL,
-	E_CAL_SOURCE_TYPE_LAST
+	E_CAL_SOURCE_TYPE_WEATHER,
+	E_CAL_SOURCE_TYPE_LAST,
 } ECalSourceType;
 
 /* Set mode status for the e_cal_set_mode function */
Index: calendar/libedata-cal/e-data-cal-factory.c
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/libedata-cal/e-data-cal-factory.c,v
retrieving revision 1.9
diff -u -r1.9 e-data-cal-factory.c
--- calendar/libedata-cal/e-data-cal-factory.c	2 Aug 2004 11:38:08 -0000	1.9
+++ calendar/libedata-cal/e-data-cal-factory.c	25 Sep 2004 09:13:02 -0000
@@ -69,6 +69,8 @@
 		return ICAL_VTODO_COMPONENT;
 	case GNOME_Evolution_Calendar_TYPE_JOURNAL:
 		return ICAL_VJOURNAL_COMPONENT;
+	case GNOME_Evolution_Calendar_TYPE_WEATHER:
+		return ICAL_XWEATHER_COMPONENT;
 	}
 	
 	return ICAL_NO_COMPONENT;
Index: calendar/libical/src/libical/icalcomponent.c
===================================================================
RCS file: /cvs/gnome/libical/src/libical/icalcomponent.c,v
retrieving revision 1.27
diff -u -r1.27 icalcomponent.c
--- calendar/libical/src/libical/icalcomponent.c	21 Oct 2003 18:28:29 -0000	1.27
+++ calendar/libical/src/libical/icalcomponent.c	25 Sep 2004 09:13:03 -0000
@@ -1254,6 +1254,7 @@
     { ICAL_XDAYLIGHT_COMPONENT, "DAYLIGHT" }, /*but are not really components*/
     { ICAL_X_COMPONENT, "X" },
     { ICAL_VSCHEDULE_COMPONENT, "SCHEDULE" },
+    { ICAL_XWEATHER_COMPONENT, "X-WEATHER" },
 
     /* CAP components */
     { ICAL_VCAR_COMPONENT, "VCAR" },  
Index: calendar/libical/src/libical/icalenums.h
===================================================================
RCS file: /cvs/gnome/libical/src/libical/icalenums.h,v
retrieving revision 1.9
diff -u -r1.9 icalenums.h
--- calendar/libical/src/libical/icalenums.h	11 Sep 2003 22:04:19 -0000	1.9
+++ calendar/libical/src/libical/icalenums.h	25 Sep 2004 09:13:03 -0000
@@ -60,8 +60,9 @@
     ICAL_VCAR_COMPONENT,
     ICAL_VCOMMAND_COMPONENT,
     ICAL_XLICINVALID_COMPONENT,
-    ICAL_XLICMIMEPART_COMPONENT /* a non-stardard component that mirrors
-				structure of MIME data */
+    ICAL_XLICMIMEPART_COMPONENT, /* a non-stardard component that mirrors
+				 structure of MIME data */
+    ICAL_XWEATHER_COMPONENT
 
 } icalcomponent_kind;
 
Index: calendar/libical/src/libicalss/icalssyacc.h
===================================================================
RCS file: /cvs/gnome/libical/src/libicalss/icalssyacc.h,v
retrieving revision 1.12
diff -u -r1.12 icalssyacc.h
--- calendar/libical/src/libicalss/icalssyacc.h	28 May 2004 10:14:24 -0000	1.12
+++ calendar/libical/src/libicalss/icalssyacc.h	25 Sep 2004 09:13:03 -0000
@@ -78,7 +78,7 @@
 typedef union YYSTYPE {
 	char* v_string;
 } YYSTYPE;
-/* Line 1249 of yacc.c.  */
+/* Line 1240 of yacc.c.  */
 #line 82 "icalssyacc.h"
 # define yystype YYSTYPE /* obsolescent; will be withdrawn */
 # define YYSTYPE_IS_DECLARED 1
Index: libedataserver/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/libedataserver/Makefile.am,v
retrieving revision 1.16
diff -u -r1.16 Makefile.am
--- libedataserver/Makefile.am	23 Sep 2004 16:45:29 -0000	1.16
+++ libedataserver/Makefile.am	25 Sep 2004 09:13:19 -0000
@@ -75,4 +75,4 @@
 EXTRA_DIST = 						\
 	$(pkgconfig_DATA:-$(BASE_VERSION).pc=.pc.in)
 
-DISTCLEANFILES = $(pkgconfig_DATA)
\ No newline at end of file
+DISTCLEANFILES = $(pkgconfig_DATA)
Index: servers/groupwise/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/servers/groupwise/Makefile.am,v
retrieving revision 1.10
diff -u -r1.10 Makefile.am
--- servers/groupwise/Makefile.am	23 Sep 2004 16:45:29 -0000	1.10
+++ servers/groupwise/Makefile.am	25 Sep 2004 09:13:19 -0000
@@ -58,4 +58,4 @@
 
 EXTRA_DIST = $(pkgconfig_DATA:-$(BASE_VERSION).pc=.pc.in)
 
-DISTCLEANFILES = $(pkgconfig_DATA)
\ No newline at end of file
+DISTCLEANFILES = $(pkgconfig_DATA)
Index: src/GNOME_Evolution_DataServerLDAP.server.in.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/GNOME_Evolution_DataServerLDAP.server.in.in,v
retrieving revision 1.12
diff -u -r1.12 GNOME_Evolution_DataServerLDAP.server.in.in
--- src/GNOME_Evolution_DataServerLDAP.server.in.in	11 Mar 2004 10:23:24 -0000	1.12
+++ src/GNOME_Evolution_DataServerLDAP.server.in.in	25 Sep 2004 09:13:19 -0000
@@ -37,6 +37,7 @@
 		<item value="webcal"/>
 		<item value="groupwise"/>
 		<item value="contacts"/>
+		<item value="weather"/>
         </oaf_attribute>
 </oaf_server>
 
Index: src/GNOME_Evolution_DataServerNOLDAP.server.in.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/GNOME_Evolution_DataServerNOLDAP.server.in.in,v
retrieving revision 1.14
diff -u -r1.14 GNOME_Evolution_DataServerNOLDAP.server.in.in
--- src/GNOME_Evolution_DataServerNOLDAP.server.in.in	11 Mar 2004 10:23:24 -0000	1.14
+++ src/GNOME_Evolution_DataServerNOLDAP.server.in.in	25 Sep 2004 09:13:19 -0000
@@ -36,6 +36,7 @@
 		<item value="webcal"/>
 		<item value="groupwise"/>
 		<item value="contacts"/>
+		<item value="weather"/>
         </oaf_attribute>
 </oaf_server>
 
Index: src/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/Makefile.am,v
retrieving revision 1.16
diff -u -r1.16 Makefile.am
--- src/Makefile.am	23 Sep 2004 16:45:29 -0000	1.16
+++ src/Makefile.am	25 Sep 2004 09:13:19 -0000
@@ -67,6 +67,7 @@
 	$(top_builddir)/calendar/backends/http/libecalbackendhttp.la		\
 	$(top_builddir)/addressbook/libebook/libebook-1.2.la			\
 	$(top_builddir)/calendar/backends/contacts/libecalbackendcontacts.la	\
+	$(top_builddir)/calendar/backends/weather/libecalbackendweather.la	\
 	$(LDAP_LIBS)								\
 	$(E_DATA_SERVER_LIBS)
 
Index: src/server.c
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/server.c,v
retrieving revision 1.20
diff -u -r1.20 server.c
--- src/server.c	2 Aug 2004 18:26:18 -0000	1.20
+++ src/server.c	25 Sep 2004 09:13:19 -0000
@@ -56,6 +56,7 @@
 #include <backends/groupwise/e-cal-backend-groupwise.h>
 #include <backends/http/e-cal-backend-http.h>
 #include <backends/contacts/e-cal-backend-contacts.h>
+#include <backends/weather/e-cal-backend-weather.h>
 
 #include "server-interface-check.h"
 #include "server-logging.h"
@@ -239,6 +240,7 @@
 	e_data_cal_factory_register_method (e_data_cal_factory, "webcal", ICAL_VEVENT_COMPONENT, E_TYPE_CAL_BACKEND_HTTP);
 	e_data_cal_factory_register_method (e_data_cal_factory, "webcal", ICAL_VTODO_COMPONENT, E_TYPE_CAL_BACKEND_HTTP);
 	e_data_cal_factory_register_method (e_data_cal_factory, "contacts", ICAL_VEVENT_COMPONENT, E_TYPE_CAL_BACKEND_CONTACTS);
+	e_data_cal_factory_register_method (e_data_cal_factory, "weather", ICAL_XWEATHER_COMPONENT, E_TYPE_CAL_BACKEND_WEATHER);
 
 	if (!e_data_cal_factory_register_storage (e_data_cal_factory, E_DATA_CAL_FACTORY_OAF_ID)) {
 		bonobo_object_unref (BONOBO_OBJECT (e_data_cal_factory));
